cc-pk-service/src/routes/pkServiceRoutes.ts

```diff
@@ -14,33 +15,39 @@ pkRouter.get('/system/ping', (_:Request, res:Response) => {
 
 // Route which returns user's items
 // http://localhost:5123/v1/users/:usedId/items
-pkRouter.get('/users/:userId/items', async (req:Request, res:Response) => {
-  if (req.params.userId === userId) {
-    const userSurveys:Survey[] = await getSurveys();
-    const reusableItems:Item[] = await getReusables(userSurveys);
+pkRouter.get('/users/:userId/items', async (req:Request, res:Response, next:NextFunction) => {
+  req.header('Token');
+  try {
+    const token:any = req.headers.token ?? '';
```

Comment:
Maybe change to req.headers.authorization, then split the token as "Bearer" is left in it. Sent you code snippets to check.